projects/core/components/notification/notification.component.ts

```diff
@@ -16,10 +16,12 @@ import {
     TuiNotificationDefaultOptions,
 } from '@taiga-ui/core/tokens';
 import {TuiNotificationT} from '@taiga-ui/core/types';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
 import {Observable} from 'rxjs';
 
 @Component({
-    selector: 'tui-notification',
+    selector:
+        'tui-notification,[tuiNotification],[tuiNotification][tuiNotificationChevron]',
```

Comment:
if I'm not mistaken, we wanted the chevron to appear on the interactive element (`button[tuiNotification]`, `a[tuiNotification]`). Without extra selector `tuiNotificationChevron`

Reply:
fixed